### DISCUSSION TITLE
Request history support

### DISCUSSION THREAD

**Discussion by gigson** [mockoon/mockoon#447] [Ideas] [answered]
Hello,
I just want to suggest a new feature - Request history. Just a tab where we would be able to see requests that came in and how Mockoon responded. I think this would be a very helpful feature for debugging.

**Comment by 255kb** [comment, marked as answer]
Hi @gigson, this feature is already there, but we know it's a little bit hidden: https://mockoon.com/docs/latest/requests-logging/
We hope it will fulfill your needs 😊
